src/modules/csm/types.py

```diff
@@ -19,7 +19,7 @@ class ReportData:
     consensusVersion: int
     ref_slot: SlotNumber
     tree_root: HexBytes
-    tree_cid: CID | Literal[""]
+    tree_cid: CID
```

Comment:
`prev_cid` still can be `Literal[""]` and hence zeroed first report will result in `tree_cid = ""`.